templates/zerver/help/change-your-language.md

```diff
@@ -1,37 +1,42 @@
 # Change your language
 
-By default, the Zulip UI is displayed in English.  However, Zulip has
-full support for internationalization, and the UI has been translated
-into many other languages.  You can see the Zulip UI in other
-languages temporarily by modifying the URLs using the appropriate
-language code, e.g. `/ru/` for Russian.
+By default, the Zulip UI is displayed in English.  However, Zulip has full
```

Comment:
The first sentence here is not that useful, I think, so removed.
The second sentence doesn't make sense to a non-programmer, so removed.